### PR TITLE
meeting and topic status change implemented

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 1]
```diff
@@ -112,7 +153,7 @@ angular.module('dashboard')
                             }
                         }
                         userPersonGuid = self.meeting.userPersonGuid;
-                        lastEventId = self.meeting.lastEventId; // 19734, 20281;
+                        lastEventId = 20281; // self.meeting.lastEventId; // 19734, 20281;
```
Comment (tkenakka) [inline app/scripts/controllers/meeting.status.ctrl.js]: Remove debug code

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 2]
```diff
@@ -35,6 +35,48 @@ angular.module('dashboard')
...
+                        changedTopicGuidArray.push(topic.Proposal.topicGuid);
+                    }
+                }, changedTopicGuidArray);
+
+                for (var j = 0; j < self.meeting.topicList.length; j++) {
+                    var topic = self.meeting.topicList[j];
+                    if (angular.isObject(topic)) {
+                        topic.isModified = (changedTopicGuidArray.indexOf(topic.topicGuid) > CONST.NOTFOUND);
+                        if (!isMobile && angular.equals(topic.topicGuid, selectedTopicGuid)) {
+                            StorageSrv.setKey(CONST.KEY.TOPIC, angular.copy(topic));
```
Comment (tkenakka) [inline app/scripts/controllers/meeting.status.ctrl.js:73]: Why is selected Topic set on proposal status change?

Comment (evidentti) [inline app/scripts/controllers/meeting.status.ctrl.js:73]: meeting.html uses topic to show isModified state
<span ng-show="ctrl.topic.isModified">

**Comment by tkenakka** [comment]
LGTM if we disregard the conflict.

**Pull request merged**